spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -614,6 +615,42 @@ public Resource getResource() {
 			 */
 			private boolean buffered = true;
 
+			/**
+			 * Check for log file existence so it can be recreated it if an external
+			 * process/agent has renamed it.
+			 */
+			private boolean checkExists = false;
+
+			/**
+			 * If the value returned from ServletRequest.getAttribute(conditionIf) yields
+			 * a null value, logging of the request will be skipped.
+			 */
+			private String conditionIf;
+
+			/**
+			 * If the value returned from ServletRequest.getAttribute(conditionUnless)
+			 * yields a non-null value, the logging of the request will be skipped.
+			 */
+			private String conditionUnless;
+
+			/**
+			 * Character set used by the log file. If it is <code>null</code>, the system
+			 * default character set will be used. An empty string will be treated as
+			 * <code>null</code> when this property is assigned.
+			 */
+			private String encoding;
+
+			/**
+			 * Use IPv6 canonical representation format as defined by RFC 5952 in output.
+			 */
+			private boolean ipv6Canonical = false;
+
+			/**
+			 * Set the locale used to format timestamps in log entries and in log file
+			 * name suffix.
+			 */
+			private String locale = Locale.getDefault().toString();
```

Comment:
We can't really do that as the sole purpose of the field init is to express a default value that isn't dependant of the environment. I'll remove that as part of the polish.